cerberus-metrics-topic/README.md

```diff
@@ -0,0 +1,62 @@
+# Cerberus Metrics Topic
+
+This template creates an SNS topic that is used by various components to publish Cerberus metrics to, if enabled.
```

Comment:
'if enabled' is awkward.  How about 'This optional template ... that can be used by various components ...' or similar?